namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/analytical-platform-irsa.tf

```diff
@@ -0,0 +1,98 @@
+module "ap_irsa" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-irsa?ref=2.0.0"
+
+  # EKS configuration
+  eks_cluster_name = var.eks_cluster_name
+
+  # IRSA configuration
+  service_account_name = "hmpps-contacts-to-ap-s3"
+  namespace            = var.namespace
+  role_policy_arns = {
+    s3 = aws_iam_policy.ap_policy.arn
+  }
```

Comment:
I would remove this file from the initial build - we haven't talked to the AP team (as far as I know) so will have no jobs to export data to their S3 bucket.  We can always add this later.

Reply:
removed this file